### PR TITLE
Add new rulesets

### PR DESCRIPTION
**Pull request by Bisaloo**
New rulesets added:

- mozillazine-fr.org
- mk2.com

### CODE REVIEW

File: src/chrome/content/rules/mk2.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<ruleset name="MK2">
+  <target host="mk2.com" />
+  <target host="www.mk2.com" />
+  <target host="m.mk2.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/mk2.xml]: Please also add api.mk2.com and comments for refused media.mk2.com and mismatched preprod.mk2.com

File: src/chrome/content/rules/mozillazine-fr.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<ruleset name="mozillazine-fr (partial)">
```
Comment (J0WI) [inline src/chrome/content/rules/mozillazine-fr.xml]: For what reason is this exposed as partial?

File: src/chrome/content/rules/mozillazine-fr.xml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+	<target host="mozillazine-fr.org"/>
+	
+	<exclusion pattern="^http://(www\.)?cname.mozillazine-fr\.org" />
+	<test url="http://cname.mozillazine-fr.org"/>
```
Comment (J0WI) [inline src/chrome/content/rules/mozillazine-fr.xml]: This is not necessary as long as the url is not a target host. But please add comments for non-working hosts.

**Comment by J0WI** [comment]
Thanks for your contribution.
Please separate unrelated hosts in different PRs next time.

**Comment by Bisaloo** [comment]
I made the requested changes. Thank you for your comments, let me know if there is something else I should change.

**Comment by J0WI** [comment]
Yes, you should mention that `www.mozillazine-fr.org` is also mismatched.

**Comment by Bisaloo** [comment]
Ok, it should be better now.

**Pull request merged**